starter-core/src/main/java/io/micronaut/starter/feature/awslambdacustomruntime/templates/awsCustomRuntimeReadme.rocker.raw

```diff
@@ -0,0 +1,10 @@
+## Deployment with GraalVM
+
+If you want to deploy to AWS Lambda as a GraalVM native image, run:
+
+```bash
+./gradlew buildNativeLambda -Pmicronaut.runtime=lambda
```

Comment:
@ilopmar `-Pmicronaut.runtime=lambda` should not be necessary right?

Reply:
It is necessary if you create an application and by default you have `runtime("netty")` so you can start your application normally but then want to deploy it to AWS Lambda + GraalVM